### PR TITLE
Updated dependencies, fixed a TypeError

### PR DESCRIPTION
**Pull request by txiklerr**
Updated all dependencies, and fixed a TypeError at './controllers/authcontroller.ts'.

### CODE REVIEW

**Comment by mraible** [comment]
Thanks @ChiKler! We need to update the [blog post](https://developer.okta.com/blog/2020/09/14/deno-with-auth) for this example before we can merge. I'll do that next week.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 .vscode
-.env
\ No newline at end of file
+.env
+
+run.bat
```
Comment (alisaduncan) [inline .gitignore:4]: A `run.bat` file isn't used in the repo, so would mind reverting this change, please?

File: controllers/authcontroller.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { Router } from 'https://deno.land/x/opine@0.12.0/mod.ts';
-import { config } from "https://deno.land/x/dotenv/mod.ts";
+import { Router } from 'https://deno.land/x/opine@1.9.0/mod.ts';
```
Comment (alisaduncan) [inline controllers/authcontroller.ts:1]: I'm having trouble running these changes, but it works for me using the latest opine. Can you please change the dependency version for opine to `opine@2.0.0` for all places where we import opine?

**Review by alisaduncan — COMMENTED** [review]
Thank you for this PR @ChiKler and for alerting us to some problems you ran into! I tried applying these changes to verify and had an issue with the opine version.

I added some comments for requested changes and after they are resolved, we can merge this PR. Thanks again!

**Comment by alisaduncan** [comment]
Thank you for this PR @ChiKler and for alerting us to problems with out-of-date dependencies. I've made updates to this code example and the blog post to address them.

I'm closing this PR in favor of #3. Thank you again!

**Pull request closed without merging**